code-challenges.js

```diff
@@ -20,42 +20,77 @@ const fruit4 = "kiwi"
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 // Pseudo code:
+// Create a function named "temp" that takes the boiling point and makes it an argument to return defined outcomes "is at boiling point", "is below boiling point" and "is above boiling point".  Our Boiling point = 212
```

Comment:
Make sure to include your logic.  Pseudocode is meant to be a roadmap on how you plan to solve the problem.  In this case, I see conditional statements in your code, this should be included in your pseudocode as well.

Reply:
Pseudocode should show on line 23.  I am not sure why it didn't push on the original push.